mail/admin.py

```diff
@@ -0,0 +1,18 @@
+"""
+Admin site bindings for mail
+"""
+
+from django.contrib import admin
+
+from .models import BlockedEmailRegex
+
+
+class BlockedEmailRegexAdmin(admin.ModelAdmin):
+    """Admin for BlockedEmailRegex"""
+
+    model = BlockedEmailRegex
+    list_display = ["match", "created_on", "updated_on"]
+    search_fields = ["match"]
```

Comment:
Can we add hint text in the admin with example values? I'm not sure whether I would use "*@example.com" or "\w+@example.com" to ban a whole domain.

Reply:
How's this for a few examples?

<img width="981" alt="Screen Shot 2020-07-14 at 4 45 41 PM" src="https://user-images.githubusercontent.com/187676/87474777-a852a480-c5f1-11ea-98fd-5ae0c2e2e899.png">